lang/cs/Org.Apache.REEF.Utilities/Attributes/TestingAttribute.cs

```diff
@@ -15,9 +15,16 @@
 // specific language governing permissions and limitations
 // under the License.
 
-namespace Org.Apache.REEF.Common.Services
+using System;
+
+namespace Org.Apache.REEF.Utilities.Attributes
 {
-    public interface IService
+    /// <summary>
+    /// Attribute target should only be used in tests.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.All)]
+    public sealed class TestingAttribute : Attribute
```

Comment:
It just indicates to other developers that certain functions should only be used in the test setting. The classes I'm using the attribute in is internal anyway, so it is not visible to users.